### PR TITLE
Issue #3004540 by truls1502: Artifact issue on Travis tests (*.1 - *.3)

### PR DESCRIPTION
**Pull request by truls1502**
## Problem
Getting errors on Travis tests *.1-*.3 such as https://travis-ci.org/SignWo/open_social/builds/437319033 

More detailed is explained on drupal.org' issue.

## Solution
The Artifact is only for internal use, so the solution is to skip the process regarding Artifact for external users/forked git so we can still use the same .travis.yml. 

## Issue tracker
- https://www.drupal.org/project/social/issues/3004540

## HTT
- [ ] Check out the code changes
- [ ] Login as user X and do this and this and that
- [ ] Notice it doesn't work
- [ ] Checkout to this branch
- [ ] Try this and this and that and see that it works now

## Release notes
<describe the release notes>

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -76,4 +76,4 @@ script:
   - if [ "$TEST_SUITE" = "accessibility" ]; then docker exec -it social_ci_web bash /var/www/scripts/social/check-accessibility.sh install; fi
 
 after_success:
-  - bash scripts/social/trigger_dockerhub.sh
+  - if [ "$TRAVIS_PULL_REQUEST_SLUG" == "goalgorilla/open_social" ]; then bash scripts/social/trigger_dockerhub.sh
```
Comment (ronaldtebrake) [inline .travis.yml]: Make sure you end your statement correctly so travis doesnt end 

Also see why you have to do it like below:
https://www.thegeekstuff.com/2010/06/bash-if-statement-examples

`if [ "$TRAVIS_PULL_REQUEST_SLUG" == "goalgorilla/open_social" ]; then bash scripts/social/trigger_dockerhub.sh; fi`

Comment (truls1502) [reply .travis.yml]: Thank you @ronaldtebrake. This one I might have overlooked when committed even I copied and paste in. However, there is a new commit on way.

**Comment by truls1502** [comment]
It worked fine in your repo, but still failing on my forked-repo https://travis-ci.org/SignWo/open_social/builds/439570520 😕

**Comment by ronaldtebrake** [comment]
> It worked fine in your repo, but still failing on my forked-repo https://travis-ci.org/SignWo/open_social/builds/439570520 😕

Could you try restarting it? It looked like composer require goalgorilla/open_social tried to find the last commit but it wasn't able to download it yet hence the:

`[InvalidArgumentException]                                                   
  Could not find package goalgorilla/open_social in a version matching dev-fi  
  x/travis-artifacts#54c275c7bfb7bbab75fe182e49512ae1e632a607    `

**Comment by truls1502** [comment]
@ronaldtebrake - I have restarted three times that time I committed and again today two times. Still no luck 😕

**Comment by ronaldtebrake** [comment]
By the looks of it your branch is not available on packagist see the long list of branches on the right side:
https://packagist.org/packages/goalgorilla/open_social 

So it's not able to download it. 
Maybe you can try to alter you composer.json to let it get the code from your local git repo instead of it being downloaded from packagist.

**Comment by truls1502** [comment]
I can see the problem. I will try to make modify Travis again ;)

**Comment by truls1502** [comment]
Is there something I have overlooked again at `.travis.yml` since I am still unluck on https://travis-ci.org/SignWo/open_social/builds/440710471?

**Comment by ronaldtebrake** [comment]
It still seems to try and upload the artifacts, maybe you can echo a couple of variables and try to debug it from there?

**Comment by jaapjan** [comment]
@truls1502 https://github.com/goalgorilla/open_social_scripts

**Comment by jaapjan** [comment]
Closing PR for now. As it only costs resources and there is still work needed here.

**Pull request closed without merging**